packages/repository/src/repository-mixin.ts

```diff
@@ -62,6 +62,18 @@ export function RepositoryMixin<T extends Class<any>>(superClass: T) {
         .tag('repository');
     }
 
+    /**
+     * Retrieve the repository instance from the given Repository class
+     *
+     * @param repo The repository class to retrieve the instance of
+     */
+    // tslint:disable-next-line:no-any
+    async getRepository<Repo extends Repository<any>>(
```

Comment:
Okay then I'm for using `Model` here and fine with it being imported as `RepositoryMixin` is used. My reason being VS code intelliSense would be able to figure it out and we already have to import `juggler` and `DataSourceConstructor` etc. for `RepositoryMixin` anyway.

Reply:
This does mean that the scope of this PR would get **much** larger since I'd have to go into the docs to make sure that the imports are there. @strongloop/lb-next-dev thoughts before I make this change?

Reply:
Ah on that note, if this is something we're doing in our codebase already (losing type safety because of `any`), we can create an issue to track it and address it separately.